airbase/pom.xml

```diff
@@ -181,7 +181,7 @@
         <dep.logback.version>1.4.14</dep.logback.version>
         <dep.modernizer.version>2.7.0</dep.modernizer.version>
         <dep.opentelemetry.version>1.34.1</dep.opentelemetry.version>
-        <dep.opentelemetry-instrumentation.version>1.32.0</dep.opentelemetry-instrumentation.version>
+        <dep.opentelemetry-instrumentation.version>2.0.0</dep.opentelemetry-instrumentation.version>
```

Comment:
What's the impact of this update? It seems like a major change based on the version bump.

Reply:
The breaking change is:

```
The default OTLP protocol has been changed from grpc to http/protobuf in order to align with the [specification](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.28.0/specification/protocol/exporter.md#specify-protocol). You can switch to the grpc protocol using OTEL_EXPORTER_OTLP_PROTOCOL=grpc or -Dotel.exporter.otlp.protocol=grpc.
```